frontend/src/types/index.ts

```diff
@@ -0,0 +1,3 @@
+import { TabSections, isTabSection } from "./TabSections";
+
+export { TabSections, isTabSection };

```

Comment:
it might be better just scope this to the Org view by adding the types folder in the Org directory here 
<img width="522" alt="image" src="https://github.com/user-attachments/assets/e0adc9a1-19dd-43e9-af7c-480ac3b46270">

Same for the project tab enum. 

we try to keep the files and folders as localized as possible

Reply:
moved to the org and project level